docs/packages/pkg/Fruit.rst

```diff
@@ -0,0 +1,38 @@
+.. spelling::
+
+    Fruit
+
+.. index:: testing ; Fruit
+
+.. _pkg.Fruit:
+
+Fruit
+=====
+
+-  `Official <https://github.com/google/fruit>`__
+-  `Hunterized <https://github.com/hunter-packages/fruit>`__
```

Comment:
Missed that, sorry. I've created fork: https://github.com/hunter-packages/fruit Please send patches to `hunter-3.1.1` branch. Please read https://docs.hunter.sh/en/latest/creating-new/patch.html

Reply:
https://github.com/hunter-packages/fruit/pull/1